AK/Time.h

```diff
@@ -143,6 +144,36 @@ constexpr i64 seconds_since_epoch_to_year(i64 seconds)
 }
 #endif
 
+template<typename T>
+constexpr Tuple<T, u8, u8> days_to_time(T days)
+{
+    // Adapted from https://howardhinnant.github.io/date_algorithms.html#civil_from_days
+    static_assert(NumericLimits<T>::digits() >= 20, "This algorithm has not been ported to a 16 bit signed integer");
+
+    constexpr u32 average_days_per_year = 365;
+    constexpr u32 last_day_of_year_three = 1460;
+    constexpr u32 days_in_first_hundred_era_years = 36524;
+    constexpr u64 era_days = 146097;
+    constexpr u32 last_day_of_era = era_days - 1;
+    constexpr u32 era_month_offset = 2;
+    constexpr u32 months_in_year = 12;
+
+    // Shift the epoch from 1970-01-01 to 0000-03-01.
+    days += 719468;
```

Comment:
VERYFY(days >= 0)?

Reply:
https://howardhinnant.github.io/date_algorithms.html#civil_from_days has `const Int era = (z >= 0 ? z : z - 146096) / 146097;` while this here has `T era = days / era_days;` so I assumed you assumed that `days >= 0`.